en/activity-standard/iati-activities/iati-activity/recipient-region.rst

```diff
@@ -11,19 +11,20 @@ Example ``recipient-region`` of an ``iati-activity``.
 
 | The ``recipient-region`` element can be repeated in any ``iati-activity``.
 
-| When multiple ``recipient-region`` are declared, then the ``@percentage`` values should sum to 100% for the specific ``iati-activity``.
+| When multiple ``recipient-region`` are declared, then the ``@percentage`` values should sum to 100% per vocabulary for the specific ``iati-activity``.
         
 .. code-block:: xml
 
 	<recipient-region code="489" vocabulary="1" percentage="50" />
 	<recipient-region code="389" vocabulary="1" percentage="50" />
 
-| When both the ``recipient-region`` and ``recipient-country``, then the ``@percentage`` values should sum to 100% for the specific ``iati-activity``.
+| When both the ``recipient-region`` and ``recipient-country``, then the ``@percentage`` values should sum to 100% per region vocabulary for the specific ``iati-activity``.
```

Comment:
Same as [this comment](https://github.com/IATI/IATI-Extra-Documentation/pull/559/files#r280082386). I suppose this should say:

> When both the ``recipient-region`` and ``recipient-country`` **are declared**, then […]